src/Knp/Bundle/KnpBundlesBundle/Resources/views/layout.html.twig

```diff
@@ -32,6 +32,11 @@
 <body{% block body_attributes %}{% endblock %}>
 
 <header id="page-header">
+
+     <div id="bannerConsent">
+         A new version of KNP Bundles is coming! Contact us here if you want to help and be a beta tester Contact KNP Labs. => if you want to help out and become <a href="http://knplabs.com/fr/contact" target="_blank">Contact KNP Labs</a>.
```

Comment:
Isn't there an issue with this sentence ?

Reply:
what do you mean?

Reply:
Doesn't seem gramatically correct :
`if you want to help out and become Contact KNPLabs`

Reply:
@lwagner an idea ?

Reply:
I've update this

Reply:
`A new version of KNP Bundles is coming! <a href="http://knplabs.com/fr/contact" target="_blank" rel="noopener noreferrer">Contact KNPLabs</a> if you want to become a beta tester.`

Don't forget the `rel` attribute in case of `target="_blank"`